### PR TITLE
rubocopify most parts of the application controller

### PR DESCRIPTION
**Pull request by tessi**
This is just formatting to make our code base more consistent.

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -68,13 +68,14 @@ def handle_unverified_request
   def params_filter
     self.utf8nize!(params) if RUBY_VERSION >= '1.9'
   end
+
   def utf8nize!(obj)
```
Comment (myabc) [inline app/controllers/application_controller.rb]: @tessi Perhaps merge #1109 first?

Comment (tessi) [inline app/controllers/application_controller.rb]: yup. but our internal ci is not green atm (and travis is still working on this PR) - so i can't merge #1109 atm

**Comment by tessi** [comment]
[PullReview stats](https://www.pullreview.com/github/opf/openproject/reviews/fix/app_controller_refactorings)

**Comment by mfrister** [comment]
:+1:

**Pull request merged**